persistence/cahe_store.go

```diff
@@ -23,9 +23,15 @@ type CacheStore interface {
 	// whether the key was found.
 	Get(key string, value interface{}) error
 
+	// Mget retrieves a list of items, return in reference array values.
+	Mget(values []interface{}, keys ...string) error
```

Comment:
Why add this to the interface, especially if you're not implementing it in all the concrete types?
Please apply this to all the funcs that match this condition in the entire PR.  Just add them to redis.go